### PR TITLE
fix(components): update button padding on icon button

### PR DESCRIPTION
**Pull request by chris-at-jobber**
## Changes

Reduced left and right padding on icon button so that these buttons have a square shape rather than a slightly-wider-than-tall rectangular shape

### Changed

- reduced left and right padding on `hasIcon` button that does not also have a label

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Button/Button.css [hunk 1]
```diff
@@ -140,7 +140,14 @@
 }
 
 /* Icon */
+
+.hasIcon {
+  width: 2.25rem;
```
Comment (seanhealy) [inline packages/components/src/Button/Button.css:145]: Can we make this based off of one of our spacing units?

Comment (seanhealy) [inline packages/components/src/Button/Button.css:145]: Can we make this based off of one of our spacing units?

File: packages/components/src/Button/Button.tsx [hunk 2]
```diff
@@ -95,6 +95,7 @@ export function Button(props: ButtonProps) {
   } = props;
 
   const buttonClassNames = classnames(styles.button, styles[size], {
+    [styles.hasIcon]: icon,
```
Comment (JeffMarvin) [inline packages/components/src/Button/Button.tsx:98]:
```suggestion
    [styles.onlyIcon]: icon && !label,
```

File: packages/components/src/Button/Button.css.d.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export const disabled: string;
 export const small: string;
 export const base: string;
 export const large: string;
+export const hasIcon: string;
```
Comment (JeffMarvin) [inline packages/components/src/Button/Button.css.d.ts:13]:
```suggestion
export const onlyIcon: string;
```

File: packages/components/src/Button/Button.css [hunk 4]
```diff
@@ -140,7 +140,14 @@
 }
 
 /* Icon */
+
+.hasIcon {
```
Comment (JeffMarvin) [inline packages/components/src/Button/Button.css:144]:
```suggestion
.onlyIcon {
```

**Comment by JeffMarvin** [comment]
Talked to @chris-at-jobber and verified the approach I suggest. Approving and merging.

**Pull request merged**